### PR TITLE
Meaningless values for {model_name}_cpl_dt in nuopc.runconfig 1deg_jra55do_iaf

### PR DESCRIPTION
**Pull request by ezhilsabareesh8**
This PR address [#132](https://github.com/COSIMA/access-om3/issues/132) by setting meaningless values for `{model_name}_cpl_dt` in `nuopc.runconfig` .

### CODE REVIEW

**Review by dougiesquire — COMMENTED** [review]
Thanks @ezhilsabareesh8. I assume you tried a negative number, e.g. -999, and it didn't work (that's my recollection)? Did you look into whether we can add comments?

**Comment by ezhilsabareesh8** [comment]
Yes negative numbers cause the `ESMF_TimeIntervalSet` routine to hang up during execution.

I just figured out that we can add comments in the `nuopc.runconfig` file by including `#` symbol, will update them shortly.

File: nuopc.runconfig [hunk 1]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 3600
+     atm_cpl_dt = 99999     #{model}_cpl_dt is not used as it is overridden by the overall time step in nuopc.runseq
```
Comment (dougiesquire) [inline nuopc.runconfig:281]: Could we add to every line something like `not used in ACCESS-OM3`

Comment (anton-seaice) [reply nuopc.runconfig:281]: Maybe just say "#not used" and people can do a git blame for details ?

Comment (ezhilsabareesh8) [reply nuopc.runconfig:281]: For all the unused variables?

Comment (anton-seaice) [reply nuopc.runconfig:281]: I guess so (we are talking about 7 lines) ?

Comment (ezhilsabareesh8) [reply nuopc.runconfig]: Added comments for all cpl time steps in this commit ec0b83b

**Review by dougiesquire — APPROVED** [review]
LGTM

**Review by dougiesquire — CHANGES_REQUESTED** [review]
Actually, could we remove all the additional spaces, e.g. `atm_cpl_dt = 99999 #not used`. It would nice to have consistent styling across different configs (so that we can do diffs etc) and a single space before the commit is probably simplest

**Comment by dougiesquire** [comment]
In the TWG meeting today, we agreed to set `ocn_cpl_dt` to the coupling timestep and all other `<component>_cpl_dt = 99999`. We decided this was the clearest approach that still works with `stop_option = nsteps`. @ezhilsabareesh8 would you be able to update this PR?

**Comment by ezhilsabareesh8** [comment]
> In the TWG meeting today, we agreed to set `ocn_cpl_dt` to the coupling timestep and all other `<component>_cpl_dt = 99999`. We decided this was the clearest approach that still works with `stop_option = nsteps`. @ezhilsabareesh8 would you be able to update this PR?

Updated in this commit [5b8acda](https://github.com/COSIMA/MOM6-CICE6/pull/55/commits/5b8acda185474afe4af48062210d53b0dc44c232)

File: nuopc.runconfig [hunk 2]
```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 99999     #not used
+     atm_cpl_dt = 99999 #not used
      calendar = GREGORIAN
      end_restart = .false.
      glc_avg_period = yearly
      glc_cpl_dt = 86400
```
Comment (minghangli-uni) [inline nuopc.runconfig:285]: `glc_cpl_dt = 99999 #not used` instead of 86400

Comment (ezhilsabareesh8) [reply nuopc.runconfig]: updated in this commit adaf65c

**Review by dougiesquire — APPROVED** [review]
Thanks @ezhilsabareesh8 et al. LGTM

**Comment by dougiesquire** [comment]
Please squash and merge to clean up the commit history

**Pull request merged**